Codeinwp/neve
# Opt-in toggle on about page

### Summary
Adds opt-in toggle to about sidebar. Related with https://github.com/Codeinwp/ti-about-page/pull/32 which has been already merged. 


### Will affect visual aspect of the product

NO

### Test instructions


- Go to about page, there should be a toggle there
- Test if it works fine


Closes Codeinwp/neve-pro-addon#261.
